src/ut-stubs/utstub-helpers.c

```diff
@@ -191,10 +191,12 @@ void UT_CheckForOpenSockets(void)
         {
             id = i;
             UT_FIXUP_ID(id, UT_OBJTYPE_QUEUE);
-            UtAssert_Failed("UT_Queue %d left open. ID=%x\n", (int)i, (unsigned int)id);
             ++InUse;
```

Comment:
Recommend just removing the `InUse` counter here.  This was probably a remnant from a previous code structure that printed a single failure at the end of the loop, but then the assert was moved to show each ID that was left open.    There isn't much value in just printing an increment counter.